duckduckgo.pot

```diff
@@ -7560,6 +7599,11 @@ msgstr ""
 msgid "More in %s"
 msgstr ""
 
+# smartling.placeholder_format=C
+msgctxt "Ads GDPR, internal use only. Do not change"
+msgid "More info"
+msgstr ""
+
```

Comment:
So, what happened here is that we've added `msgctxt "Ads GDPR, internal use only. Do not change"` to this new string. This confused the test script into looking for the expected translations and thus the failure. I don't quite know why we're adding a new string when an existing "More info" already exists (literally the next block down here 🙂). I suggest using the existing string and maybe just changing the `msgctxt` to be more generic if you feel squeamish about re-using something for `precise_user_location`. I would avoid duplication and I would certainly avoid re-using the `msgctxt` for internal use unless there's a specific requirement I'm not aware of. If that is the case, please ping me in the `~localizers` channel on Mattermost. Thanks! cc @franfaccin